### ISSUE TITLE
Non-Interactive installation on Docker

### ISSUE THREAD

**Issue by specio** [intel/confidential-computing.tee.dcap.pccs#14]
Hi since version 1.6 I am running PCCS  on dockerized environment.
Recently I had to update from 1.8.100.2 to 1.10.103.1 due to TCB recovery.

I used to install it with non-interactive command:
echo "N" | sudo apt install sgx-dcap-pccs -y
but now I have to handle another prompt about possible data loss when upgrading.

I managed to use:
{ echo "Y"; echo "N"; } | sudo apt install sgx-dcap-pccs -y
but it doesn't work anymore...

it fails with:
```

Preparing to unpack .../sgx-dcap-pccs_1.10.103.1-bionic1_amd64.deb ...
Unpacking sgx-dcap-pccs (1.10.103.1-bionic1) ...
Setting up sgx-dcap-pccs (1.10.103.1-bionic1) ...
Checking nodejs version ...
nodejs is installed, continue...
[1;33mWarning: If you are upgrading PCCS from an old release, the existing cache database will be updated automatically. [0m 
[1;33m         It's strongly recommended to backup your existing cache database first and then continue the installation. [0m 
[1;33m         For DCAP releases 1.8 and earlier, the cache database can't be updated so you need to delete it manually. [0m 
dpkg: error processing package sgx-dcap-pccs (--configure):
 installed sgx-dcap-pccs package post-installation script subprocess returned error exit status 1
Errors were encountered while processing:
 sgx-dcap-pccs

```

**Comment by zhuyifeiRuichuang** [comment]
I also encountered a TCB-related error message. This was caused by a time issue. To resolve it, ensure that all machines in the current environment have their time zones and system times synchronized, and connect them to an NTP server. If there is no internet access, connect to an internal NTP server instead. If deploying in containers, configure the environment variable `-e TZ=<timezone>` to specify the time zone, and make sure the container’s time matches the physical machine’s time.

**Comment by mtomczew** [comment]
Non-interactive installation with `DEBIAN_FRONTEND=noninteractive` environment variable is still supported in PCCS version 1.25. 
Interactive installation definitely has more steps and questions than when this issue is created and is more difficult to automate.

I'll close this old issue. If the problem with non-interactive installation occur in one of recent PCCS releases, please create a new ticket.
